include/highfive/H5Object.hpp

```diff
@@ -51,12 +79,40 @@ class Object {
     hid_t _hid;
 
   private:
+    // Init with an low-level object id
+    Object(hid_t);
```

Comment:
```suggestion
    explicit Object(hid_t);
```
To avoid explicit conversion